project1/styles.css

```diff
@@ -0,0 +1,86 @@
+blockquote p {
+    font-family: 'Satisfy', cursive;
+    font-size: 30px;
+    line-height: 1;
+    margin-left: 150px;
+    margin-right: 150px;
+    text-align: center;
+}
+
+body {
+
+    color: white;
+}
+
+div {
+    font-size: 40px;
+    text-align: center;
+}
+
+h1 {
+    margin-bottom: 0;
+}
+
+a {
+    margin-top: 0
+}
+
+header {
+    color: navy;
+    font-family: 'Fjalla One', sans-serif;
+    font-size: 60px;
+    letter-spacing: 3px;
+
+}
+
+h1 {
+    margin-top: 50px;
+    margin-bottom: 10px;
+}
+
+p {
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 300;
+    line-height: 1.75;
+}
+
+section p {
```

Comment:
If you still think the spacing around your bullets looks wonky, it is likely because the margins for the top of your paragraphs doesn't match the bottom margins. Even though you never set a `margin-top` here, there's a browser default of 15px, which you can see in the Layout tab in dev tools:

![Screen Shot 2019-09-24 at 3 08 14 PM](https://user-images.githubusercontent.com/2653980/65542470-3f246800-dedd-11e9-92ac-31bce95f8e31.png)

You can target _just_ the first `p` element inside each section by using a **pseudo-class** -- we'll learn more about these soon, but here it would look like this:

```
section p:first-of-type {
    margin-top: 10px;
}
```

You would add this on line 54, _after_ your current `section p` ruleset.

Reply:
I didn't end up needing to do this one, the other suggestion worked, but I will keep this in mind! Looking forward to learning more about this